### ISSUE TITLE
Some issues/queries with new mapcache system and db2sql

### ISSUE THREAD

**Issue by EyesOfAHawk** [HerculesWS/Hercules#1978]
### Issue Prelude

-  [x] I have not modified the source prior to reproducing this issue.
-  [x] I am using the latest version of Hercules.
-  [x] I am aware that this report will be closed or deleted if it becomes obvious that I am stating the false.

### Description

Some things I have found with the new mapcache system.
- When I use `./map-server --convert-old-mapcache`, it reads item, mob and skill dbs, as if I also used db2sql.
- Need some document for the command, unclear how to use, especially in relation to the data folder. Can you still have your map files in the /data folder to build maps? Cause before I was using mapcache.exe on my Windows, with my maps in the Hercules/data folder to create map_cache.dat, then committing map_cache.dat to be used on Linux.
- In relation to db2sql, it gives me an error if I have a custom script command being used in it.
```
[Error]: script error in file 'item_db2.conf' item ID 22360
    parse_line: expect command, missing function name or calling undeclared function
    aura(.@aura, 0);
```

**Branch(es):**
- [x] master

### Operating System

CentOS 7

**Comment by Emistry** [comment]
>  I have not modified the source prior to reproducing this issue.
Actually you have edited the source hahaha

Make sure the `aura` script command are added to the source and build/compile successfully.

**Comment by EyesOfAHawk** [comment]
Nope...aura is a command in a plugin. And it is a script command only, does not have any other edits. Therefore I have not edited source.

**Comment by MishimaHaruna** [comment]
I don't understand what mapcache has to do with db2sql. Is this about two separate issues? (if so, please open two separate tickets, so that both things can be tracked separately)

As for the second question you asked: there should have been absolutely no changes with regard to where the files are read from, since the new plugin uses the same methods used before by the standalone executable. It is still subject to the grfio configuration (`conf/grf-files.txt`).

**Comment by EyesOfAHawk** [comment]
The issue is that db2sql runs when I run the mapcache command.

**Comment by MishimaHaruna** [comment]
Ok, I'll try to reproduce it.

In any case, I recommend against adding the plugins you don't want to run every time (such as db2sql, mapcache, itemdb2doc, naviluagenerator, etc) to plugin.conf. The recommended way to run those those "developer's" plugins is to pass their name to the map server's command line. For example:

```
./map-server --load-plugin db2sql --db2sql
```

or

```
./map-server --load-plugin mapcache --convert-old-mapcache
```

That way you never risk running functions from a plugin you don't need. Plugins that define custom commands, or anything that is necessary every time you run the server, can still go into plugins.conf.
